notes/2024-02-06 biglep feedback for PR #65.md

```diff
@@ -0,0 +1,99 @@
+# Purpose
+This is intended to accompany https://github.com/ipld/github-mgmt/pull/65.
+I did it as its own file to enable threaded replies on specific lines/sections of this file with the PR UI.
+If I didn't do this, we'd be limited to quote replies in https://github.com/ipld/github-mgmt/pull/65.
+Given that I expect a decent amount of back-and-forth on some of these suggestions, I figured quote replies would get unweildy.
+I'm not assuming this file will get checked in.  It's purely to help with the code review process.
+Maybe at the end if there is useful content here, we copy/paste it into the PR itself.  (Alternatively we could merge it in.)
+Or I guess we could just move into Github discussions?  (I didn't do that because wanted to keep all the context here.)
+Obivously I could have done this in another tool like Google Docs, but I wanted to keep the discussion more open and discoverable (especially since this will the basis for changes to libp2p, ipfs, etc. repos).
+If I should do something else, please let me know!
+
+# General github-mgmt feedback/wishes
+## Documentation I wish was more readily accessible when using github-mgmt
```

Comment:
2024-02-08 conversation: @biglep to extract this out to a generic github-mgmt backlog issue.  It's not required for this permission cleanup campaign.